liquibase_changelog/patternatlas_warm-starting-patterns-tags.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.6.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.6.xsd">
+    <changeSet author="Felix Truger" id="1710330017868-3">
```

Comment:
pls set this to some (higher) value than the id in [liquibase_changelog/patternatlas_warm-starting-patterns.xml](https://github.com/PatternAtlas/pattern-atlas-content/pull/35/files#diff-6908cc893f5ada03ddb2d7cfd944648a1f000b40783b92473de6af4274d1ddc7). I'm not 100% sure but liquibase might complain if there are two changesets with the same id